pkg/sql/exec_util.go

```diff
@@ -1025,6 +1027,16 @@ func (q *queryMeta) cancel() {
 	q.ctxCancel()
 }
 
+// getStatement returns a cleaned version of the query associated
+// with this queryMeta.
+func (q *queryMeta) getStatement() (string, error) {
+	parsed, err := parser.ParseOne(q.rawStmt)
+	if err != nil {
+		return "", err
```

Comment:
Maybe we could include the error in the message. "error getting statement: err"

Reply:
and then not return an error at all?

Reply:
Since we can return an error, I'm not sure if this is needed.

Reply:
but the thing we do above is just completely bail out. So that means that if a single active session has some kind of broken corrupted statement, we can get none of the active statements at all. I don't like that very much! When we receive this error in the `serialize` method can we somehow just keep on keeping on, so we have resilience to bugs in show sessions?

Reply:
ok, that makes sense.